content/shared/influxdb3-cli/update/database/_index.md

```diff
@@ -0,0 +1,84 @@
+The `influxdb3 update database` command updates an existing database in your {{< product-name >}} instance.
+
+Use this command to update a database's retention period.
+
+## Usage
+
+<!--pytest.mark.skip-->
+
+```bash
+influxdb3 update database [OPTIONS] --database <DATABASE_NAME>
+```
+
+## Arguments
+
+- **`DATABASE_NAME`**: (Required) The name of the database to update. Valid database names are alphanumeric and start with a letter or number. Dashes (-) and underscores (_) are allowed.
```

Comment:
I don't think the naming restrictions are important here; only in the creation process. When a user is updated the database, the database is already named.

```suggestion
- **`DATABASE_NAME`**: (Required) The name of the database to update.
```